README.md

```diff
@@ -1 +1,25 @@
-# github-flow
\ No newline at end of file
+Project Name: Hello Microverse
+
+This is an example of github flow. Create new feature branch and make commits
+
+Built With
+HTML
+
+
+Authors
+👤 Author1 Franco Gonzalez Colloricchio
+
+GitHub: @francoaugusto
+Twitter: @franncoaugusto
+LinkedIn: https://www.linkedin.com/in/franco-gonzalez-9b732024a/
+
+
+Acknowledgments
+
+Hat tip to anyone whose code was used
+Inspiration
+etc
+
+
```

Comment:
- [ ] Good start with your readme :+1:  but I think you can add some main section to make it more descriptive and professional. I'm sharing with you a good example [here](https://github.com/codecaiine/hello-microverse/blob/main/README.md) based on [this template](https://github.com/microverseinc/readme-template) Customize it by adding the main section you need and fill it with your information. Please fix it :smile: